content/blog/2019/05/2019-05-11-docs-sig-announcement.adoc

```diff
@@ -0,0 +1,41 @@
+---
+:layout: post
+:title: "Jenkins Documentation Special Interest Group"
+:tags:
+- documentation
+- docs
+- community
+:author: markewaite
+:sig: docs
+---
+
+image:/images/logos/256.png[Jenkins, role=center, float=right]
+
+We're pleased to announce the formation of the Jenkins Documentation Special Interest Group.
+The Docs SIG will encourage, create, and review documentation contributions and improve documentation with contributors and external communities.
```

Comment:
```suggestion
The Docs SIG encourages contributors and external communities to create and review Jenkins documentation. 
```
See if this works for you.  The existing sentence is a bit obtuse and doesn't parse well so I fiddled with it.  You may be able to do better.